src/types.ts

```diff
@@ -0,0 +1,33 @@
+import { CumulusMessage, ReplaceConfig } from '@cumulus/types/message';
+import { Context } from 'aws-lambda';
+import { ChildProcessWithoutNullStreams } from 'child_process';
+
+export interface CumulusMessageAdapterError { stderrBuffer: string }
+
+export interface InvokeCumulusMessageAdapterResult {
+  cmaProcess: ChildProcessWithoutNullStreams,
+  errorObj: CumulusMessageAdapterError
+}
+
+export interface LoadNestedEventInput {
+  input: unknown,
+  config: unknown,
+  messageConfig?: unknown
+}
+export interface CumulusMessageWithPayload extends CumulusMessage {
+  payload: {
+    granules?: { granuleId: string }[] | undefined
+  }
+  meta: {
+    stack?: string,
+    input_granules: { granuleId: string }[] | undefined
+  }
+}
+export interface CMAMessage {
+  cma: {
+    event?: object
+  }
+  replace?: ReplaceConfig
+}
+
+export type TaskFunction = (...args: [LoadNestedEventInput, Context]) => undefined;
```

Comment:
```suggestion
export type TaskFunction = (...args: [LoadNestedEventInput, Context]) => void;
```

Reply:
Why `void` here?

Reply:
Because, as defined, that function isn't returning anything.

From https://www.typescriptlang.org/docs/handbook/basic-types.html#void

"You may commonly see this as the return type of functions that do not return a value"

Reply:
This: https://github.com/nasa/cumulus-message-adapter-js/commit/b4ac934173bc23b91edd0e80abfa7f851900188c wound up using this for a different purpose (and the one I *thought* it was being used for when I asked that).    I think it *should* be expected to return an `undefined` type now.